waspc/cli/Command/Telemetry.hs

```diff
@@ -9,11 +9,19 @@ import           Data.Maybe                (isJust)
 import qualified System.Environment        as ENV
 
 import           Command                   (Command, CommandError (..))
+import           Command.Common           (waspSaysC)
 import qualified Command.Call
 import           Command.Telemetry.Common  (ensureTelemetryCacheDirExists)
 import qualified Command.Telemetry.Project as TlmProject
 import qualified Command.Telemetry.User    as TlmUser
 
+telemetry :: Command ()
+telemetry = do
+    isTelemetryDisabled <- liftIO $ isJust <$> ENV.lookupEnv "WASP_TELEMETRY_DISABLE"
+    waspSaysC $ "Telemetry is currently: " <> (if isTelemetryDisabled
```

Comment:
Looking good! I wonder, why use `<>` and not `++`? I get that `<>` is more general, but I somehow always default to `++`. Keep in mind I am not suggesting we have to change it, just wondering how do you make that decision. I think I go with `++` because I knew about it before I learned about `<>`, which is not a very convincing reason :D.

Reply:
I don't have a convincing reason either. I just thought it's something I could apply and I did.